### PR TITLE
Added new values for subcategory (traffic)

### PR DESCRIPTION
**Pull request by aqueralt**
Added the values "pedestrianOnRoad", "bikerOnRoad", and "tramApproaching" in Alert subCategory, for alerts related to traffic events.

### CODE REVIEW

**Comment by albertoabellagarcia** [comment]
Could you provide the use case in the ADOPTER.yaml file? 
https://github.com/smart-data-models/dataModel.Alert/blob/master/Alert/ADOPTERS.yaml
Thanks

**Comment by albertoabellagarcia** [comment]
Thank you very much but specs are generated automatically so, you do not need to PR these documents, once accepted they are updated. 
You only need to PR on the schema.json (and in the ADOPTERS.yaml)
Thanks

File: Alert/doc/spec.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Entity: Alert
```
Comment (albertoabellagarcia) [inline Alert/doc/spec.md:5]: PR on this file is rejected

File: Alert/doc/spec_DE.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Entität: Alert
```
Comment (albertoabellagarcia) [inline Alert/doc/spec_DE.md:5]: PR on this file is rejected

File: Alert/doc/spec_ES.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Entidad: Alerta
```
Comment (albertoabellagarcia) [inline Alert/doc/spec_ES.md:5]: PR on this file is rejected

File: Alert/doc/spec_FR.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ Entité : Alerte
```
Comment (albertoabellagarcia) [inline Alert/doc/spec_FR.md:5]: PR on this file is rejected

File: Alert/model.yaml [hunk 5]
```diff
@@ -287,6 +287,9 @@ Alert:
             - roadWorks
```
Comment (albertoabellagarcia) [inline Alert/model.yaml:287]: PR on this file is rejected

File: Alert/schema.json [hunk 6]
```diff
@@ -19,7 +19,7 @@
       "properties": {
```
Comment (albertoabellagarcia) [inline Alert/schema.json:19]: ok

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
I'm going to accept the changes but not through this PR. 
sorry for the job you did on documents that are automatically generated. We've modified our process to include a note about not doing it in the header of specifications. 
Like this [https://github.com/smart-data-models/dataModel.RoboticIndustrialActivities/blob/master/Pallet/doc/spec.md](https://github.com/smart-data-models/dataModel.RoboticIndustrialActivities/blob/master/Pallet/doc/spec.md)

**Comment by albertoabellagarcia** [comment]
As explained before changes are accepted but PR no.
thanks

**Comment by albertoabellagarcia** [comment]
Updated

**Comment by aqueralt** [comment]
Many thanks, and sorry for the inconveniences. I will update the Adopters file asap.

**Pull request closed without merging**